content/rest-api/troubleshooting.md

```diff
@@ -0,0 +1,83 @@
+# Troubleshooting guide
+
+This guide describes the most common errors you can experience when using the API and the solutions to fix it.
+
+## Missing client id
+
+Two problems can be the cause of the following response when authenticating to the API:
+
+```
+{
+  "code": 422,
+  "message": "Parameter "client_id" is missing or does not match any client, or secret is invalid"
+}
+```
+
+### Base 64 not correctly encoded
+
+You are not correctly encoding client and secret in base 64.
+We encourage you to use the [Postman collection](/getting-started-postman-collection.html) or the [PHP client](/php-client/introduction.html), that handle it automatically for you.
+
+If you use the command line to generate the base 64, please ensure that you do it this way:
+
+```
+echo -n "client_id:secret" | base64
+```
+
+Do note the option `-n` to avoid to print the trailing newline character and encode it.
+
+If you still experience the same error, please follow the second solution.
+
+### Apache strip the authentication header
+
+If you are sure to provide the correct base 64 of the client and secret, it probably means that Apache is not correctly set up.
+Various Apache modules can strip the authorization header “Authorization: Basic base64client_id:secret”.
+
+Add the following line in your virtual host file:
+
+```
+SetEnvIf Authorization "(.*)" HTTP_AUTHORIZATION=$1
+```
+
+## Redirection on connection page
+
+When requesting the API, you are redirected to the API, with any endpoints.
+
+This problem when the project has been updated from version 1.6.
+It is coming from the security file `app/config/security.yml` in the PIM project.
+
+The declaration order of the keys is important in this file.
```

Comment:
If you want to get to a new line after this sentence, you'll have to put 2 spaces at the end of the line.